org.openscenegraph.android/jni/JNIUtils.h

```diff
@@ -37,6 +38,51 @@ class ReferencedType: public T, public osg::Referenced
 protected:
 	~ReferencedType() { }
 };
+typedef ReferencedType<osg::Vec2> RefVec2;
 typedef ReferencedType<osg::Vec3> RefVec3;
 typedef ReferencedType<osg::Vec4> RefVec4;
 typedef ReferencedType<osg::Quat> RefQuat;
+
+inline osg::Vec4 Interpolate(osg::Vec2 _texcoord, osg::Image* _img)
```

Comment:
I agree Jordi: for efficiency reasons, it can be "const osg::Vec2 &texcoord" - I just needed this function quickly, so I wasn't that picky with the reference. The underscoring: noted and will be changed.